backend/controllers/cellMove/viewCellSharingAssessmentDetails.js

```diff
@@ -0,0 +1,37 @@
+const moment = require('moment')
+const { serviceUnavailableMessage } = require('../../common-messages')
+const { putLastNameFirst, hasLength } = require('../../utils')
+const { getBackLinkData } = require('./cellMoveUtils')
+
+module.exports = ({ elite2Api, logError }) => async (req, res) => {
+  const { offenderNo } = req.params
+
+  try {
+    const { firstName, lastName, assignedLivingUnit } = await elite2Api.getDetails(res.locals, offenderNo, true)
+
+    const assessments = await elite2Api.getAssessments(res.locals, { offenderNumbers: [offenderNo], code: 'CSR' })
```

Comment:
The offender details endpoint does not return the approval date or the agency Id, so calling an assessment specific endpoint to get all the data. Can be changed to the details page in the future if we update the API.